### DISCUSSION TITLE
Can I suggest a project to add in the `Used by` section?

### DISCUSSION THREAD

**Discussion by aarontravass** [typicode/husky#1373] [Q&A]
Was wondering if you guys are open to suggestions.

**Comment by typicode** [comment]
Since there's a lot of project using husky we can't add all of them but feel free to suggest :)

**Reply by aarontravass** [reply]
[tinyhttp](https://github.com/tinyhttp/tinyhttp) - It's a typescript first web framework built as a replacement to express. It's smaller and faster. 

(P.S. - I'm a co-maintainer)

**Reply by typicode** [reply]
Hmm 🤔 I think I've heard about it 😉
https://github.com/typicode/json-server/blob/main/package.json#L42

 I'll add it :)

**Reply by typicode** [reply]
Oops messed up the link to the dependency but you probably saw that json-server is using it :)

**Reply by aarontravass** [reply]
Thanks a lot. I appreciate it!
